data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -135,22 +120,22 @@ public void getLineageInfo() throws ParseException, SemanticException {
 		ogw.startWalking(topNodes, null);
 	}
 
-//	private ASTNode search(int type, ASTNode node) {
-//		System.out.println("searching node = " + node);
-//		if (node != null) {
-//			if (node.getType() == type) {
-//				return node;
-//			} else {
-//				for (int i = 0; i < node.getChildCount(); i++) {
-//					ASTNode foundNode = search(type, (ASTNode) node.getChild(i));
-//					return foundNode;
-//				}
-//			}
-//		} else {
-//			return null;
-//		}
-//		return null;
-//	}
+/*	private ASTNode search(int type, ASTNode node) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)